chorus_book/src/guide-transport.md

```diff
@@ -8,25 +8,46 @@ ChoRus provides two built-in transports: `local` and `http`.
 
 ### The Local Transport
 
-The `local` transport is used to execute choreographies on the same machine on different threads. This is useful for testing and prototyping.
-
-To use the `local` transport, import the `LocalTransport` struct from the `chorus_lib` crate.
+The `local` transport is used to execute choreographies on the same machine on different threads. This is useful for testing and prototyping. Each `local` transport is defined over `LocalTransportChannel`, which contains the set of `ChoreographyLocation` that the `local` transport operates on. You can build a `LocalTransportChannel` by importing the `LocalTransportChannel` stsruct from the `chorus_lib` crate.
 
 ```rust
 # extern crate chorus_lib;
-use chorus_lib::transport::local::LocalTransport;
+# use chorus_lib::core::{ChoreographyLocation};
+# use chorus_lib::{LocationSet};
+# #[derive(ChoreographyLocation)]
+# struct Alice;
+# #[derive(ChoreographyLocation)]
+# struct Bob;
+use chorus_lib::transport::local::LocalTransportChannel;
+
+let transport_channel = LocalTransportChannel::new().with(Alice).with(Bob);
 ```
 
-You can construct a `LocalTransport` instance by passing a slice of locations to the `from` method.
+To use the `local` transport, first import the `LocalTransport` struct from the `chorus_lib` crate.
```

Comment:
This sentence seems redundant.

Reply:
Well, we have this sentence in the current version of the [book](https://lsd-ucsc.github.io/ChoRus/guide-transport.html): "To use the `local` transport, import the `LocalTransport` struct from the `chorus_lib` crate." 
In this commit, it's just in a different paragraph. But if you do think that it shouldn't have been there in the first place, sure! I'll delete it.